email-validator.inc.php

```diff
@@ -67,8 +67,6 @@ function checkHostName( &$strHostName )
 	{
 		if ( preg_match( "/^[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}$/", $strHostName ) )
 			$numHostIP = @gethostbyaddr ( $strHostName ) ;
-		else if ( $this -> countChars( $strHostName, "." ) == 1 )
```

Comment:
This assumes, that all domains always have a 'www' subdomain configured. In my case that wasn't / isn't the case (even though it might not be smart). But the absence of a 'www' CNAME or A record should not be abused as an indicator for spammyness I believe.